src/TypeDefinition.js

```diff
@@ -381,6 +381,7 @@ export type NavigationTransitionProps = {
   // is the index of the scene
   scene: NavigationScene,
   index: number,
+  navigation: NavigationScreenProp<NavigationRoute, NavigationAction>,
```

Comment:
@ericvicenti how do you see the `_getChildNavigation` going forward? Shall we stick to it and pass it around, or remove it entirely and just stick to `addNavigationHelpers`?